_data/glossary.yml

```diff
@@ -0,0 +1,31 @@
+canonical-tags:
+- Fundamentals
+- K8s Objects
```

Comment:
@chenopis would "API Object" suffice as a label? I think it might be useful to be able to look at a term and know that it is built into the Kubernetes API (so this label would be separate from "API", which would include things like "apiserver"